service/rating.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'file_handler'
+
+# Description/Explanation of Rating module
+module Rating
```

Comment:
Please, refactor it, currently you execute top_ids each time outside module and pass it in each function, it should be only inside